.github/workflows/add-missing-labels/add-missing-labels.js

```diff
@@ -0,0 +1,29 @@
+const { getAddedComponentPackages, scriptPrefix } = require('./get-added-component-packages');
+
+module.exports = async ({ github, context }) => {
+    const packagePrefix = '@justeattakeaway/';
+
+    try {
+        const addedComponentPackages = await getAddedComponentPackages('packages/components/pie-webc/package.json', packagePrefix);
+
+        if (addedComponentPackages.length===0) {
+            console.error(`${scriptPrefix} No components were added, nothing to update.`);
+        } else {
+            for await (const componentName of addedComponentPackages) {
+                const name = componentName.replace(packagePrefix, '');
+
+                console.info(`${scriptPrefix} Adding label for the package "${name}"...`);
+
+                await github.rest.issues.createLabel({
+                    name,
+                    color: 'ededed',
```

Comment:
I wonder if there is a way to randomise the colour 🤣

Reply:
Might be rather easy... I thought all components were supposed to follow the same gray for consistence 🤔

Reply:
It's possible, but I'm glad we're going with the consistent colour because all of the other components use this one 😄